data/mcl1/01_protein/crd/protein.pdb

```diff
@@ -70,2475 +70,2620 @@ REMARK 350   BIOMT2  1   0.000000 1.000000 0.000000   0.000000
 REMARK 350   BIOMT3  1   0.000000 0.000000 1.000000   0.000000
 REMARK 888
 REMARK 888 WRITTEN BY MAESTRO (A PRODUCT OF SCHRODINGER, LLC)
-SEQRES   1 B  151  GLY ASP GLU LEU TYR ARG GLN SER LEU GLU ILE ILE SER
-SEQRES   2 B  151  ARG TYR LEU ARG GLU GLN ALA THR GLY ALA LYS ASP THR
-SEQRES   3 B  151  LYS PRO MET GLY ARG SER GLY ALA THR SER ARG LYS ALA
-SEQRES   4 B  151  LEU GLU THR LEU ARG ARG VAL GLY ASP GLY VAL GLN ARG
-SEQRES   5 B  151  ASN HIS GLU THR ALA PHE GLN GLY MET LEU ARG LYS LEU
-SEQRES   6 B  151  ASP ILE LYS ASN GLU ASP ASP VAL LYS SER LEU SER ARG
-SEQRES   7 B  151  VAL MET ILE HIS VAL PHE SER ASP GLY VAL THR ASN TRP
-SEQRES   8 B  151  GLY ARG ILE VAL THR LEU ILE SER PHE GLY ALA PHE VAL
-SEQRES   9 B  151  ALA LYS HIS LEU LYS THR ILE ASN GLN GLU SER CYS ILE
-SEQRES  10 B  151  GLU PRO LEU ALA GLU SER ILE THR ASP VAL LEU VAL ARG
-SEQRES  11 B  151  THR LYS ARG ASP TRP LEU VAL LYS GLN ARG GLY TRP ASP
-SEQRES  12 B  151  GLY PHE VAL GLU PHE PHE HIS VAL
-HELIX    1   1 ASP B  172  ALA B  190  1
-HELIX    2   2 GLY B  203  ASN B  223  1
-HELIX    3   3 GLU B  225  LEU B  235  1
-HELIX    4   4 GLU B  240  LYS B  244  1
-HELIX    5   5 SER B  247  PHE B  254  1
-HELIX    6   6 TRP B  261  THR B  280  1
-HELIX    7   7 GLU B  288  LYS B  308  1
-HELIX    8   8 GLY B  311  PHE B  318  1
+SEQRES   1 B  153  ACE GLY ASP GLU LEU TYR ARG GLN SER LEU GLU ILE ILE
```

Comment:
Is the ACE cap necessary? I was under the impression that there were missing residues on the C termini but not the N one? (although you did mention some truncation in the assay? is this why it's needed?)

Reply:
I don't know if I'm missing something here that's specific for this system, but we do expect the proteins to be capped in order to run the simulations, don't we?

Reply:
> we do expect the proteins to be capped in order to run the simulations, don't we?

Depends on the approach and system. I believe we agreed on "cap if there are missing residues, don't cap if there aren't" when it comes to dealing with termini (somewhere around https://github.com/openforcefield/protein-ligand-benchmark/pull/52#issuecomment-1185414491). Hence why MCL-1 didn't have caps until this PR (the paper is a little bit complicated and I'm not even sure if my interpretation is correct) - @bobym probably has a better record of the approach taken here.

Reply:
It confuses me that Uniprot indicates EDELY as the starting sequence, but the supplementary information of the Friberg et al. paper starts with GDELY... But yeah the construct seems to start at that glycine?

Also do we have any idea if the WT or Δ5 constructs were used for the FPA assays?

Reply:
Welcome to my confusion... Here's where I'm at with this:

> Also do we have any idea if the WT or Δ5 constructs were used for the FPA assays?

The Δ5 construct was used for the assays.

> I don't know if I'm missing something here that's specific for this system, but we do expect the proteins to be capped in order to run the simulations, don't we?

> @bobym probably has a better record of the approach taken here.
 
I've capped MCL-1 because (even though they used the truncated protein in assay), the best chain of pdb MHW3 still has two unmodeled residues at the C-terminus. 

> It confuses me that Uniprot indicates EDELY as the starting sequence, but the supplementary information of the Friberg et al. paper starts with GDELY... But yeah the construct seems to start at that glycine?

The glycine is an artifact from the expression tag that was not able to be cleaved off.

Reply:
@jchodera : Agreed, we should definitely be clearer in the paper about this. As it currently stands, I've been going with this:

1. Does the crystal structure include the full length of the protein used in the assay? If yes, do not cap. If no, cap it.
2. Are the termini of the crystal structure fully modeled?  If yes, do not cap. If no, cap it.

Other questions to answer are:

- What do you do if one chain in a dimer/multimer is fully modeled (and not needing a cap) but the other is not?
- What if one terminus is has structural data, but the other does not?

> In cases where we don't have structural data for the missing terminal residues, suggesting the terminal residues are disordered for that terminus, we would truncate and cap.

In this instance, the C-terminal residues are not resolved, necessitating the cap. 

> I'm not sure I understand the motivation to cap both termini if one terminus corresponds to the assay construct boundary. Is this just a limitation of Schrodinger?

This is a limitation of Schrodinger, which would cap both N-and C-termini. That being said, it was my understanding that generally if you cap one you would want to cap the other to allow the protein to be completely capped + neutral. (Unless the resolved, charged terminus is involved in important interactions.)

Reply:
I suppose the protein picks up a net charge if you only cap one terminus, but the Schrodinger limitation appears to be the bigger problems, so capping both is less problematic than not capping either.

I like the idea of examining the charged terminus for important interactions---that should probably be a warning in the paper. In SARS-CoV-2 Mpro (not part of this benchmark), this actually happens! The charged N-terminal Ser1 NH3+ coordinates with the Glu166 sidechain from the opposite chain, as well as some backbone carbonyls. Capping this could compromise the flexibility.

Reply:
> I like the idea of examining the charged terminus for important interactions

@jchodera: Agreed. I think we also might want to think about examining whether charged termini are important for any inter-chain/inter-monomer interactions that could destabilize a structure.